org/wildfly/undertow/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.undertow
+
+DESCRIPTION: Undertow entry point capability, main service
+
+REGISTERED BY:
+  
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: false
+
+PRIVATE: false
```

Comment:
This needs to be PRIVATE: true until we can rework this into a proper capability API.  UndertowService is not suitable.

I'm not saying other code in WF can't use this for now. They are already using UndertowService, so they are already "hacking into private API". I just mean this can't be declared non-private until we have the right contract.

The undertow extension itself can use UndertowService if it needs API that outsiders shouldn't use. It doesn't have to use capability resolution to find internal services it needs.